lib/newspaper_works/ingest/ndnp/batch_ingester.rb

```diff
@@ -1,12 +1,43 @@
-require 'find'
 require 'date'
+require 'find'
+require 'optparse'
 
 module NewspaperWorks
   module Ingest
     module NDNP
       class BatchIngester
         attr_accessor :path, :batch
 
+        # alternate constructor from ARGV
+        # @param options [Array<String>
```

Comment:
`[Array<String>` intentional here?

Reply:
No, missing closing bracket.  Fixed in f40bf67762537a0abd84dbeadbb11144cf9972d6